mParticle-Apple-SDK/Kits/MPKitContainer.mm

```diff
@@ -1892,6 +1892,16 @@ - (void)project:(id<MPExtensionKitProtocol>)kitRegister event:(MPEvent *const)ev
     completionHandler(projectedEvents, appliedProjections);
 }
 
+- (nonnull NSArray<NSNumber *> *)configuredKitsRegistry {
```

Comment:
What's the reason for having this nonnull and returning an empty array vs what `activeKitsRegistry` does which is return nil if there's no kits? I think we should stay consistent unless there's a good reason, so in this case that would mean standardizing on what `activeKitsRegistry` is doing.

Reply:
So, I did notice this too, and went a bit back and forth on how to balance consistency vs eveything else.. Maybe this is more philosophical, but I tend to think it's almost always better to remove nullability/nillibility from the equation if you reasonably can?

I'll update it though because consistency is very important and I'm not trying to make style changes to the repo in this little PR :)

Reply:
Yeah agreed on the general concept of removing nullability, but also agree that consistency is important, so for not lets stay consistent, and later we can refactor the SDK in general to remove nullability.